extensions/wikia/CommunityPage/CommunityPage.i18n.php

```diff
@@ -311,7 +289,7 @@
 	'communitypage-anon-login' => 'Anmelden',
 	'communitypage-anon-register' => 'Anmelden',
 	'communitypage-todo-module-page-name' => 'ToDo-Liste der Community',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "communitypage-todo-module-page-name" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)